services/QuillLMS/app/queries/prompt_feedback_history.rb

```diff
@@ -0,0 +1,54 @@
+class PromptFeedbackHistory
+
+    def self.run(activity_id)
+        promptwise_postprocessing promptwise_sessions(activity_id)
+    end
+
+    def self.promptwise_sessions(activity_id)
+        activity_sessions = ActivitySession.where(activity_id: activity_id).includes(:feedback_sessions)
+        feedback_session_uids = activity_sessions.reduce([]) {|memo, a_s| memo.concat a_s.feedback_sessions.pluck(:uid)}
```

Comment:
I think in our Ruby style guide, we would wrap the argument to `concat` in parentheses (and that's also what I personally find easier to read): https://github.com/empirical-org/ruby#returns-val-parens.